class.css

```diff
@@ -28,6 +28,8 @@
 .pb3 { padding-bottom: var(--space-3) }
 .pb4 { padding-bottom: var(--space-4) }
 
+.-m2 { margin: calc(var(--space-2) * -1) }
+.-m1 { margin: calc(var(--space-1) * -1) }
```

Comment:
I gave negative margins lower precedence as to continue with the trend here. Plus this way it affords classing like `-ml1 mt-auto mb-auto` such as to support only 2 sides. Negative margins can lead to antipatterns. This makes them possible for hacks or gutters without adding too many rules.